pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupUsageAggregationTest.java

```diff
@@ -210,7 +211,10 @@ private void verifyStats(String topicString, String rgName,
                                                                 throws InterruptedException, PulsarAdminException {
         BrokerService bs = pulsar.getBrokerService();
         Awaitility.await().untilAsserted(() -> {
-            TopicStatsImpl topicStats = bs.getTopicStats().get(topicString);
+            GetStatsOptions getStatsOptions =
+                    GetStatsOptions.builder().getPreciseBacklog(false).subscriptionBacklogSize(false)
+                            .getEarliestTimeInBacklog(false).getTotalNonContiguousDeletedMessagesRange(false).build();
```

Comment:
```suggestion
                            .getEarliestTimeInBacklog(false).getTotalNonContiguousDeletedMessagesRange(true).build();
```
?

Reply:
in resourcegroup ,we do not need this data. it will use much memory ,see https://github.com/apache/pulsar/issues/18240